### DISCUSSION TITLE
Surname changes to uppercase - how to change?

### DISCUSSION THREAD

**Discussion by di6parn** [Set-OutlookSignatures/Set-OutlookSignatures#103] [Q&A] [answered]
Hello,

I am trying out the free version of Set-OutlookSignatures, and it looks like it is exactly what we need. 

While testing it, I see that the surname is all uppercase, and I can't find how to change that behaviour. I have copied and edited the **Ext frml.docx** template, but I can't find anything in the template that affects the case. 
When examining the resulting .txt and .htm files in the Signatures folder, the surname in the .txt file is formatted the way we want it (only first letter uppercase), but the .htm file has a `TEXT-TRANSFORM: uppercase` html code snippet that I would like to get rid of. How can I do that?

Thanks in advance!

Cheers,
Pär

**Comment by GruberMarkus** [comment, marked as answer]
Hi Pär,

in the Word template file "Ext frml.docx", the text "$CURRENTMAILBOXSURNAME$" has the font formatting option "all caps": Mark "$CURRENTMAILBOXSURNAME$", open the font dialog box via the dialog box launches button the Fonts menu group, "all caps" is selected. See https://support.microsoft.com/en-us/office/the-font-dialog-box-031c97f9-f969-4b4f-98f8-b14724443ab6 for details.

The template has been configured like this for demonstration purposes only. In real life, to avoid compatibility problems among different HTML renderers used by mail clients, I would suggesst a custom replacement variable converting the last name to upper case instead of relying on this specific Word/HTML feature.

Does this answer your question?

**Reply by di6parn** [reply]
Hi Markus,

Thank you, that solved it! I did change the text from "$CURRENTMAILBOXSURNAME$" to "$CurrentMailboxSurname$" thinking that would be the solution, but I (obviously) didn't think of looking at the settings in the Font dialog box.

Have a nice day!
